.github/workflows/builds.yml

```
@@ -134,4 +238,60 @@ jobs:
             BUILD_DATE=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.created'] }}
             VERSION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
             COMMIT_HASH=${{ github.sha }}
-            REVISION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.revision'] }}
\ No newline at end of file
+            REVISION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.revision'] }}
+  docker-suppression-backup-service-arm64:
+    runs-on: [self-hosted, Linux, ARM64]
+    outputs:
+      tags: ${{ steps.meta.outputs.tags }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            name=rudderstack/develop-suppression-backup-service
+            name=rudderstack/suppression-backup-service,enable=${{ github.ref == format('refs/heads/{0}', 'master') || github.event_name == 'release' }}
+          tags: |
+            type=ref,event=branch
+            type=raw,value=latest,enable=${{ github.event_name == 'release' }}
+            type=raw,value=${{ github.head_ref }}
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: rudderlabs
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/arm64
+          file: ./suppression-backup-service/Dockerfile
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}-arm64
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+            BUILD_DATE=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.created'] }}
+            VERSION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}-arm64
+            COMMIT_HASH=${{ github.sha }}
+            REVISION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.revision'] }}
+  create-manifest:
+    runs-on: ubuntu-latest
+    needs: [ docker-ent-arm64, docker-ent ]
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3.0.0
+      - name: Login to DockerHub
+        uses: docker/login-action@v2.1.0
+        with:
+          username: rudderlabs
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Create multi-arch manifest for enterprise
+        run: |
+          docker buildx imagetools create -t ${{ needs.docker-ent.outputs.tags }} ${{ needs.docker-ent-arm64.outputs.tags }} ${{ needs.docker-ent.outputs.tags }}-amd64
```

Could it be possible that `needs.docker-ent.outputs.tags` contains multiple tags and `-amd64` only get applied to the last one?

would it be more appropriate to use suffix https://github.com/docker/metadata-action?tab=readme-ov-file#flavor-input ?

```suggestion
          docker buildx imagetools create -t ${{ needs.docker-ent.outputs.tags }} ${{ needs.docker-ent-arm64.outputs.tags }} ${{ needs.docker-ent.outputs.tags }}
```